src/lib/server/locks/order-lock.ts

```diff
@@ -149,42 +150,80 @@ async function maintainOrders() {
 					case 'lightning':
 						try {
 							if (!payment.invoiceId) {
-								throw new Error('Missing invoice ID on lightning order');
+								throw new Error('Missing invoice ID on lightning payment');
 							}
-							if (payment.processor === 'phoenixd') {
-								const invoice = await phoenixdLookupInvoice(payment.invoiceId);
-
-								if (invoice.isPaid) {
-									order = await onOrderPayment(
-										order,
-										payment,
-										{
-											amount: invoice.receivedSat,
+							if (!payment.processor) {
+								throw new Error('Missing processor on lightning payment');
+							}
```

Comment:
Should add a migration in `migrations`, to add "lnd" processor to each lightning payment without processor. As in the past when there was only lnd, `processor` was not filled.

The migration file is here: https://github.com/be-BOP-io-SA/be-BOP/blob/42c59cc81b31e8894f5137c113b0aeb12d964368/src/lib/server/migrations.ts

Note for the `_id` of the migration, you can use `ObjectId.createFromTimestamp(getUnixTime(new Date("2025-05-13T08:59:51.997Z")))`.

Not sure if there are any pilots using the lnd client @Tirodem (if there are not, the migration is not needed)

Reply:
Added the migration.